packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -46,5 +46,9 @@ export const DefaultPalette: IPalette = {
   tealLight: '#00b294',
   greenDark: '#004b1c',
   green: '#107c10',
-  greenLight: '#bad80a'
+  greenLight: '#bad80a',
+  contrastBlackDisabled: '#00ff00',
```

Comment:
I will ask him to take a look at this review too. I took the colors from .scss files they were defined in. These four colors for high contrast were missing in the Glamor-based styling package palette